### DISCUSSION TITLE
Expanding sidebar

### DISCUSSION THREAD

**Discussion by morteza-ayashi** [Acode-Foundation/Acode#777] [Q&A]
![Screenshot_20230723-203734_Acode.jpg](https://github.com/deadlyjack/Acode/assets/113019884/0acd0b30-99c3-404a-83a5-6b927e4625a2)

Is there a way to expand sidebar even more? Because as you can see, in the new updates (v1.8.5 (304)), it's not enough when you open more folders. 

and if there isn't, please add a feature to expand it more, thanks.

**Comment by sebastianjnuwu** [comment]
Hello can you specify your device?

**Reply by morteza-ayashi** [reply]
Samsung Galaxy A20 With Android 11

**Reply by sebastianjnuwu** [reply]
The @deadlyjack go take a look, on mine it's working fine one thing I noticed is that the icon of the folders is different from mine and I'm in the latest version too but on an older cell phone than your j2 core android 8

![Screenshot_20230724-114908](https://github.com/deadlyjack/Acode/assets/101356556/62ff29e5-c8c5-4148-a2fd-3fe1c62ea304)

**Reply by morteza-ayashi** [reply]
It is because I installed a plugin, maybe the problem is from my phone?

**Reply by sebastianjnuwu** [reply]
I don't know how to inform only @deadlyjack

**Reply by morteza-ayashi** [reply]
OK, thanks for your help

**Reply by morteza-ayashi** [reply]
@deadlyjack Any updates?
